Scripts/LookupCSV/LookupCSV.yml

```diff
@@ -0,0 +1,31 @@
+name: LookupCSV
+script: '-'
+type: python
+tags:
+  - file
+  - csv
+  - Utility
+comment: Parses a CSV and looks for a specific value in a specific column, returning
+  a dict of the entire matching row.
+enabled: true
+args:
+  - name: entryID
+    required: true
+    description: EntryID of CSV file.
+  - name: header_row
+    description: CSV file has a header row
```

Comment:
please add optional values as it may seem that the meaning is the headers themselves,
And could you add the option to specify the headers in the arguments, as some CSV might come without it

Reply:
without the headers it still parses - do you mean allow people to arbitarily map headers? But could get a little tricky as if the user passes in less than the number of cols in the row it will result in unexpected (to them) behavior. Thoughts?

Reply:
I think that we can catch this case very easily, with a len(entered_header) == len(first_line.split(',')

But this one is not a must, your call

Reply:
This is a good idea, not a lot of extra code for a potential big boost in functionality. Added.